File: 1561. Maximum Number of Coins You Can Get/Solution.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        iRet = 0
        piles.sort()
        
        while piles:
            iRet += piles[-2]
            piles.pop()
            piles.pop()
            piles.pop(0)
            
        return iRet
        

File: 1561. Maximum Number of Coins You Can Get/Solution.cpp
class Solution {
public:
    int maxCoins(vector<int>& piles) {
        int iRet = 0;
        std::sort(piles.begin(), piles.end());
        
        for(int i=0;i<piles.size()/3;i++)
        {
            iRet += piles[piles.size()-2*i-1-1];
        }
        
        return iRet;
    }
};
